shared/actions/kbfs/index.desktop.js

```diff
@@ -5,6 +5,17 @@ import * as Constants from '../../constants/config'
 import {config} from '../../constants/types/keybase-v1'
 import type {AsyncAction} from '../../constants/types/flux'
 
+// fixup if you pass the kbfsPath in path or not
+const open = (kbfsPath: string, path: ?string) => {
```

Comment:
make this a little smarter. accept /private/chris or /keybase/private/chris and still work